cumulus_library/studies/core/documentreference.sql

```diff
@@ -72,7 +72,7 @@ WITH powerset AS (
     WHERE
         d.encounter_ref = e.encounter_ref
         AND d.status = 'current'
-        AND d.docstatus IN (NULL, 'final', 'amended')
+        AND (d.docstatus IS NULL OR d.docstatus IN ('final', 'amended'))
```

Comment:
This was a bug that happened to be caught by the new duckdb unit test. You apparently need to test for IS NULL, it doesn't work in a list like that. The new duckdb test has all these cases (as well as an `entered-in-error` case), so we're exercising this check.

Reply:
oh great catch - these inline counts i haven't been paying much attention to yet.